### PR TITLE
Improve load test workflows

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/2697
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/process-load-test-result.yml [hunk 1]
```diff
@@ -51,11 +54,11 @@ jobs:
       - name: Merge PR
         if: ${{ steps.processResults.outputs.hasPassed }}
         run: |
-          status="NOT COMPLETED"
-          while [ ! -z "$status" ]
+          checkCount="0"
+          while [ "$checkCount" != "3" ]
```
Comment (BuddhiWathsala) [inline .github/workflows/process-load-test-result.yml:58]: @dilanSachi I hope, here the 3 is referred to the number of required checks?.

Comment (dilanSachi) [reply .github/workflows/process-load-test-result.yml:58]: yes

Comment (BuddhiWathsala) [reply .github/workflows/process-load-test-result.yml:58]: Wouldn’t this loop executive forever if some checks have failed?.

Comment (dilanSachi) [reply .github/workflows/process-load-test-result.yml:58]: Added a check for that https://github.com/ballerina-platform/module-ballerina-grpc/pull/712/commits/9ace56f75b4d94f75e0d529d789da56e51791d1f

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Pull request merged**